src/main/java/com/redhat/management/approval/Request.java

```diff
@@ -163,16 +167,17 @@ private String createEncodedIdentity(RHIdentity id) {
         obj.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         String encoded = "";
 
+        Logger log = Logger.getLogger(packet.getInsightsRequestId());
```

Comment:
remove `packet`. consider to promote `log` to an instance member.

Reply:
I spent most time yesterday, tried to use instance member first. It failed due to `serializable` problem. Eventually it comes from `java.io.NotSerializableException: com.amazonaws.services.logs.AWSLogsClient`